ansible/roles/cartodb.resque/templates/cartodb_resque_worker.conf.j2

```diff
@@ -0,0 +1,18 @@
+# WARNING: This file is auto-generated by Ansible. Any changes will
+# be overwritten the next time ansible is run.
+
+description "Start a single cartodb resque worker"
+
+stop on runlevel [!2345]
+
+respawn
+# Kill respawn if more than 5 restarts in 20 mins
+respawn limit 5 20
+chdir {{ cartodb_dir }}
+
+instance $WORKERNUM
+
+script
+    RESQUEPID1={{ cartodb_resque_pid_dir }}/resque$WORKERNUM.pid
```

Comment:
This can be broken out into a top level stanza:

```bash
env RESQUEPID1={{ cartodb_resque_pid_dir }}/resque$WORKERNUM.pid
```

Unsure there is much benefit in this case though. Might be more valuable if you need to add additional stanzas later (`pre-start`, `post-stop`, etc.).